src/Tizen.Multimedia.MediaPlayer/Player/Player.cs

```diff
@@ -548,6 +548,9 @@ public virtual void Pause()
         ///     -or-<br/>
         ///     It is not able to assign the source to the player.
         ///     </exception>
+        /// <exception cref="PermissionDenied">Thrown when the permission is denied.</exception>
```

Comment:
`PermissionDenied` is just for used internally. 
I think it maybe `UnauthorizedAccessException`, right?